java-ast/addon/src/main/java/org/jboss/windup/ast/java/JavaASTProcessor.java

```diff
@@ -200,42 +204,59 @@ private JavaASTProcessor(WildcardImportResolver importResolver, CompilationUnit
         cu.accept(this);
     }
 
+    private String extractDefinitionLine(String typeDeclaration)
+    {
+        String typeLine = "";
+        String[] lines = typeDeclaration.split("\n");
+        for (String line : lines)
+        {
+            typeLine = typeLine + "\n" + line;
+            if (line.contains("{"))
+            {
+                break;
+            }
+        }
+        return typeLine;
+    }
```

Comment:
I think eclipse JDT forgets abouts some lines, so all the times I checked there were lines like "@Override public void myMethod(params) {"